src/state/service-reader.ts

```diff
@@ -34,25 +34,25 @@ export const readServices: (filePath: string) => Partial<Service>[] = (filePath:
 };
 
 const metadataLabelMapping = {
-    repoContext: "chs.local.repoContext",
+    repoContext: "chs.local.repoContext=",
```

Comment:
Is it possible to have instead a more stringent condition on [findLabel](https://github.com/companieshouse/chs-dev/blob/6e66b2f12f85788d3b93781b056e3992db41013a/src/state/service-reader.ts#L67) and have it matching a more precise `${prefix}=` rather than a "startsWith" ?

Reply:
find label is used for the traefik routes which do not have the `=` following them, I could parameterize I suppose to optionally add the `=`?

Reply:
Refactored to use a regex and work out in findLabel whether should have `=` in the prefix

Reply:
I see. I don't know exactly all the features-set for traefik, nor if in a case like [this](https://github.com/companieshouse/docker-chs-development/blob/b2cc7c05bf54bb1cbdba029a34e3bd2f68e4e620/services/modules/follow/chs-monitor-gui.docker-compose.yaml#L7-L8) 
```
      - traefik.http.routers.chs-monitor-gui.rule=Host(`follow.chs.local`)
      - traefik.http.routers.chs-monitor-gui.priority=1
```
it would be ok to return the 1st entry for a find label `traefik.http.routers.`. 
Anyhow, I think that if we can find a way to handle even traefik logic within findLabel it would be a benefit

Reply:
I think this is what is occurring now? Whereby traefik label prefixes have a trailing `.` which is used to determine whether the prefix being searched for should have a `=` or not.

Traefik via the ingress proxy is using these labels to define its routing to the service, `chs-dev` uses these to work out whether the ingress-proxy should be restarted when new services are started or not.